doc/concepts/configuration.rst

```diff
@@ -0,0 +1,651 @@
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``snapshot.dir`` and ``wal.dir`` options can be used to configure directories for storing snapshots and write-ahead logs.
+For example, you can place snapshots and write-ahead logs on different hard drives for better reliability.
+
+.. code-block:: yaml
+
+    instance001:
+      snapshot:
+        dir: '/media/drive1/snapshots'
+      wal:
+        dir: '/media/drive2/wals'
+
+To learn more about the persistence mechanism in Tarantool, see the :ref:`Persistence <concepts-data_model-persistence>` section.
+
+
+.. _configuration_run_instance:
+
+Starting Tarantool instances
+----------------------------
+
+.. _configuration_run_instance_tt:
+
+Starting instances using the tt utility
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The :ref:`tt <tt-cli>` utility is the recommended way to start Tarantool instances.
+
+Instance files or directories are organized into applications in the ``instances_enabled`` directory.
+The example below shows how a :ref:`layout <admin-start_stop_instance-multi-instance-layout>` of the application called ``app`` might look:
+
+.. code-block:: none
+
+    ├── tt.yaml
+    └── instances.enabled
+        └── app
+            ├── config.yaml
+            ├── myapp.lua
+            └── instances.yml
+
+*   ``config.yaml`` is a :ref:`configuration file <configuration_file>`.
+*   ``myapp.lua`` is a Lua script containing an :ref:`application to load <configuration_application>`.
+*   ``instances.yml`` specifies :ref:`instances <admin-start_stop_instance-multi-instance>` to run in the current environment.
+    This file might look as follows:
+
+    ..  literalinclude:: /code_snippets/snippets/replication/instances.enabled/manual_leader/instances.yml
+        :language: yaml
+        :dedent:
+
+To start all instances, use the ``tt start app`` command:
+
+    .. code-block:: console
+
+        $ tt start app
+            • Starting an instance [app:instance001]...
+            • Starting an instance [app:instance002]...
+            • Starting an instance [app:instance003]...
+
+Then, you can connect to Tarantool instances by its names using the ``tt connect`` command.
+You can learn more from the :ref:`Starting and stopping instances <admin-start_stop_instance>` section.
+
+
+
+.. _configuration_run_instance_tarantool:
+
+Starting an instance using the tarantool command
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``tarantool`` command provides additional :ref:`options <configuration_command_options>` that might be helpful for development purposes.
+Below is the syntax for starting a Tarantool instance configured in a file:
+
+..  code-block:: console
+
+    $ tarantool [OPTION ...] --name INSTANCE_NAME --config CONFIG_FILE_PATH
+
+The command below starts ``instance001`` configured in the ``config.yaml`` file:
+
+.. code-block:: console
+
+    $ tarantool --name instance001 --config config.yaml
+
+
+.. _configuration_command_options:
+
+Command-line options
```

Comment:
This is definitely not the place for `tarantool` binary options reference. Need to find place somewhere in reference or future `Tools` section and link to it.